src/index.ts

```diff
@@ -2,56 +2,53 @@ import * as $ from 'jquery';
 import 'bootstrap/js/dist/modal';
 import { showFeedback, showReputation } from './view/feedback';
 import { showEvent } from './view/event';
-import { GameController, isGameState } from './controller/GameController';
+import { GameController } from './controller/GameController';
 import { StoryEvents } from './assets/StoryEvents';
-import { CompletedEvent } from './model/Events';
+import { EndOfTurnSummary } from './model/Events';
 import { showEndScreen } from './view/endgame';
-import { US } from './scenarios/US';
+import { UK } from './scenarios/UK';
 
 $(window).on('load', () => {
+
     // Initialise game engine
     const narrative = StoryEvents;
-    const gameController = new GameController(US, narrative);
+    const gameController = new GameController(UK, narrative);
 
     // Await player response
     const onResponse = (responseId: string) => {
         // Carry out player's response
-        const competedEvent = gameController.respondToEvent(responseId);
+        const endOfTurnSummary: EndOfTurnSummary = gameController.respondToEvent(responseId);
 
         // Hide event
         $('#event-modal').modal('hide');
 
-        // Show feedback to player
-        if (competedEvent.reputation.length > 0) {
-            // Show reputation before feedback to response
-            showReputation(competedEvent.reputation);
+        // Show a reputation or just feedback
+        if (endOfTurnSummary.history.thisMonth.indicators.reputation.length > 0) {
+            // Show reputation then feedback
+            showReputation(endOfTurnSummary.history.thisMonth.indicators.reputation);
```

Comment:
Ignore. Reputations are now only shown at the end of the game, they are the end screen now.